charts/seed-bootstrap/charts/fluent-bit/values.yaml

```diff
@@ -17,3 +18,12 @@ ports:
 additionalFilters: ""
 additionalParsers: ""
 fluentBitConfigurationsOverwrites: {}
+lokiLabels:
+  kubernetesLabels:
+    container_name: "container_name"
+    namespace_name: "namespace_name"
+    pod_name: "pod_name"
+    docker_id: "docker_id"
+  systemdLabels:
+    hostname: "host_name"
+    unit: "systemd_component"
```

Comment:
Usually, CamelCase is used in YAML keys

Reply:
Yes, I know but here, the keys should match these in fluent-bit side:
https://github.com/gardener/logging/blob/de3684cfd2441de0cb5aa67573a0aa5c3237b6d7/pkg/lokiplugin/utils.go#L26